### PR TITLE
Create `rpmmacros` file

### PR DESCRIPTION
**Pull request by alfredodeza**
So that we don't end up with RPM binaries that have 'centos' in there

### CODE REVIEW

File: ansible/slave.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@
    - jenkins_user: 'jenkins-build'
   tasks:
     - name: create a jenkins-build user
-      user: name=jenkins-build comment="Jenkins Build Slave User"
+      user: name=jenkins-build comment="Jenkins Build Slave User" home="/home/{{ jenkins_user }}"
```
Comment (andrewschoen) [inline ansible/slave.yml]: Should we do ``name="{{ jenkins_user }}"`` here instead of hardcoding jenkins-build?

File: ansible/slave.yml [hunk 2]
```diff
@@ -78,10 +85,11 @@
...
       sudo: true
-      known_hosts: path='/etc/ssh/ssh_known_hosts'
-      name='docs.ceph.com'
-      # docs.ceph.com.pub is the output of `ssh-keyscan docs.ceph.com`
-      key="{{ lookup('file', 'ssh/hostkeys/docs.ceph.com.pub') }}"
+      known_hosts:
+        path: '/etc/ssh/ssh_known_hosts'
+        name: 'docs.ceph.com'
+        # docs.ceph.com.pub is the output of `ssh-keyscan docs.ceph.com`
+        key: "{{ lookup('file', 'ssh/hostkeys/docs.ceph.com.pub') }}"
```
Comment (ktdreyer) [inline ansible/slave.yml]: gah, sorry about this, thanks for fixing it.

File: ansible/files/rpmmacros.j2 [hunk 3]
```diff
@@ -0,0 +1 @@
+%dist .el{{ ansible_lsb.major_release|int }}
```
Comment (ktdreyer) [inline ansible/files/rpmmacros.j2]: can we skip the cast to an `int` here?

Comment (ktdreyer) [inline ansible/files/rpmmacros.j2]: actually `{{ ansible_distribution_major_version }}` will do it

**Pull request merged**